src/engine/bank03.asm

```diff
@@ -2808,31 +2843,44 @@ Func_d28c: ; d28c (3:528c)
 .asm_d2f5
 	ret
 
-Func_d2f6: ; d2f6 (3:52f6)
-	ld hl, $530c
+ScriptCommand_ShowSamNormalMultichoice: ; d2f6 (3:52f6)
+	ld hl, .multichoice_menu_args
 	xor a
-	call Func_d28c
-	ld a, [wd694]
+	call ShowMultichoiceTextbox
+	ld a, [wMultichoiceTextboxResult_Sam]
 	ld c, a
 	set_flag_value EVENT_FLAG_75
 	xor a
-	ld [wd694], a
+	ld [wMultichoiceTextboxResult_Sam], a
 	jp IncreaseScriptPointerBy1
 ; 0xd30c
 
-	INCROM $d30c, $d317
+.multichoice_menu_args ;d30c
+	tx SamNPCName ; NPC title for textbox under menu
+	tx Text03fe ; text for textbox under menu
+	dw SamNormalMultichoice_ConfigurationTable ; location of table configuration in bank 4
+	db $03 ; the value to return when b is pressed
+	dw wMultichoiceTextboxResult_Sam ; ram location to return result into
+	dw $0000 ; location of table containing text entries
+
 
-Func_d317: ; d317 (3:5317)
-	ld hl, $532b
-	ld a, [wd694]
-	call Func_d28c
-	ld a, [wd694]
+ScriptCommand_ShowSamTutorialMultichoice: ; d317 (s)
```

Comment:
the location here got replaced incorrectly

Reply:
Fixed in latest commit